package.json

```diff
@@ -28,6 +28,10 @@
     "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
     "link-aa-win": "node ./scripts/link-bin.js packages/amplify-app/bin/amplify-app amplify-app-dev",
     "setup-dev-win": "yarn dev-build && yarn link-win && yarn link-aa-win",
+    "update-test-data": "source ./scripts/cci-data.sh && updateTestData",
+    "update-job-data": "source ./scripts/cci-data.sh && updateJobData",
+    "process-job-data": "yarn update-job-data && ts-node ./scripts/cci-process-data.ts",
+    "workflow-results": "yarn ts-node ./scripts/cci-workflow-results.ts",
```

Comment:
These scripts can be run manually a significant change in test timings has occurred.
"process-job-data" will fetch the latest cci job data & update the cci-timings data